doc/installing.rst

```diff
@@ -0,0 +1,49 @@
+.. _installing:
+
+Installing pyRiemann
+====================
+
```

Comment:
Same here, keep pyriemann installation-related instruction short to focus on the qiskit specific part.